src/sentry/static/sentry/app/utils/theme.jsx

```diff
@@ -18,17 +18,20 @@ const theme = {
 
   blue: '#3B6ECC',
   blueLight: '#628BD6',
+  blueLightest: '#F5FAFE',
   blueDark: '#2F58A3',
 
   green: '#57be8c',
   greenLight: '#71D8A6',
+  greenLightest: '#f8fcf7',
```

Comment:
😬 what comes after?

Reply:
greenlighterestest!

jk I think the pattern is:

* colorDarkest
* colorDark
* color
* colorLight
* colorLightest

and we should cap it there. five tints per color should be enough, and if we need something lighter or darker we can consider changing one.